api/tests/LinkyLink.Tests/Service/LinksServiceTests.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Castle.Core.Configuration;
+using LinkyLink.Helpers;
+using LinkyLink.Models;
+using LinkyLink.Service;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Moq;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace LinkyLink.Tests.Service
+{
+    /// <summary>
+    /// Tests to validate methods in LinksService class.
+    /// </summary>
+    public class LinksServiceTests
```

Comment:
I tried mocking these methods but it returns 'not implemented exception'. It works fine if the these methods are not async but not otherwise. I already spent enough time to try out various approach's and decided to not include tests for that as it could be an EF issue.

LinksService.AllLinkBundlesAsync, FindLinkBundlesForUserAsync, FindLinkBundleAsync